src/WP_CLI/CommandWithDBObject.php

```diff
@@ -184,17 +184,4 @@ protected function get_formatter( &$assoc_args ) {
 		}
 		return new Formatter( $assoc_args, $fields, $this->obj_type );
 	}
-
-	/**
-	 * Given a callback, display the URL for one or more objects.
-	 *
-	 * @param array $args One or more object references.
-	 * @param string $callback Function to get URL for the object.
-	 */
-	protected function _url( $args, $callback ) {
-		foreach ( $args as $obj_id ) {
-			$object = $this->fetcher->get_check( $obj_id );
```

Comment:
The `fetcher` property didn't even exist on this abstract class, and in the extending classes it was `private`. So this code never worked. Plus it was never used, so it's safe to remove.